src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -424,6 +424,46 @@ def _qa_log_filename(self):
         """
         return self._ancillary_filename() + ".qa.log"
 
+    def _augment_measured_parameters(self, measured_parameters):
+        """
+        Augment the measured parameters dict into a dict of dicts containing the needed fields for the MeasuredParameters
+        section of the ISO XML file.
+
+        Parameters
+        ----------
+        measured_parameters : dict
+            The GeoTIFF metadata from the output product. See get_geotiff_metadata()
+
+        Returns
+        -------
+        augmented_parameters : dict
+            The metadata fields converted to a list with name, value, types, etc
+        """
+        augmented_parameters = dict()
+
+        for name, value in measured_parameters.items():
+            if isinstance(value, list):
+                value = json.dumps(value)
+
+            data_type = python_type_to_xml_type(value)
+
+            # TODO: This is hardcoded for now, but we should eventually try to guess this
```

Comment:
That was based on a conversation with Scott. Basically there are different attribute types than `processingInformation` (see the diff for the jinja2 template), but since it's the most common it's hardcoded for now.

The guessing may end up not being necessary based on the proposed changes in PR #504 as the proper type can just be added into the measured parameters YAML file.